features/recognize.js

```diff
@@ -1,11 +1,11 @@
-const { emoji, maximum, minimumMessageLength, usersExemptFromMaximum, reactionEmoji } = require('../config')
+const { maximum, minimumMessageLength, usersExemptFromMaximum, reactionEmoji } = require('../config')
 const recognition = require('../service/recognition')
 const balance = require('../service/balance')
 const winston = require('../winston')
 
 const userRegex = /<@([a-zA-Z0-9]+)>/g;
 const tagRegex = /#(\S+)/g;
-const emojiRegex = new RegExp(emoji, 'g');
```

Comment:
This RegEx seems a little more broad than I'd like, I think that custom emoji names are more restrictive than this.

If we can scope this down a little that would be my preference.

Reply:
Fixed the regex...Slack is very vauge about the requirements for a custom emoji:
![image](https://user-images.githubusercontent.com/35746073/99154270-597e4f80-2663-11eb-96f4-c15b9d4af32b.png)

So this seems to be at least a little more specific, I tested it with a couple example emojis and it looks like `'` is supported